docs/examples/1d_hist/1d_profile.py

```diff
@@ -0,0 +1,31 @@
+"""
+1d Profile
+==========
+
+Plot a 1d mean histogram (profile plot).
+"""
+
+###
+import boost_histogram as bh
+from plothist import plot_error_hist
+import matplotlib.pyplot as plt
+
+fig, ax = plt.subplots()
+
+# Regular axis with 3 bins from -1 to 1
+axis = bh.axis.Regular(3, -1, 1)
+
+data = [-0.5, -0.5, 0.0, 0.0, 0.5, 0.5]
+sample = [0, 100, 40, 60, 0, 20]
+weights = [1, 1, 1, 1, 1, 1]
```

Comment:
Should be bother with weights? They are 1 and don't contribute to the profile plot example. The easier the example the best,

Reply:
I prefer having these weights, so that the user can easily generalize the example. This is the only example in our documentation that creates an histogram without `make_hist()`, so if we remove the weights, the user will be forced to dig into the `boost_histogram` documentation to learn how to use weights.